src/commands/deploy/utils.ts

```diff
@@ -71,6 +82,85 @@ export async function getOrCreateEmptyProject(
     return { projectId: project.id, projectEnvId: projectEnv.id };
 }
 
+export async function readOrAskConfig(configPath: string): Promise<YamlProjectConfiguration> {
+    const configIOController = new YamlConfigurationIOController(configPath);
+    if (!existsSync(configPath)) {
+        const name = await readOrAskProjectName();
+
+        let region = regions[0].value;
+        if (process.env["CI"] !== "true") {
+            ({ region } = await inquirer.prompt([
+                {
+                    type: "list",
+                    name: "region",
+                    message: "Select the Genezio project region:",
+                    choices: regions,
+                },
+            ]));
+        } else {
+            log.info(
+                "Using the default region for the project because no `genezio.yaml` file was found.",
+            );
+        }
+
+        await configIOController.write({ name, region, yamlVersion: 2 });
+    }
+
+    return await configIOController.read();
+}
+
+export async function readOrAskProjectName(): Promise<string> {
+    if (existsSync("package.json")) {
+        // Read package.json content
+        const packageJson = readFileSync("package.json", "utf-8");
+        const packageJsonName = JSON.parse(packageJson)["name"];
+
+        const validProjectName: boolean = await (async () => checkProjectName(packageJsonName))()
+            .then(() => true)
+            .catch(() => false);
+
+        const projectExists = await getProjectInfoByName(packageJsonName)
+            .then(() => true)
+            .catch(() => false);
+
+        // We don't want to automatically use the package.json name if the project
+        // exists, because it could overwrite the existing project by accident.
+        if (packageJsonName !== undefined && validProjectName && !projectExists)
+            return packageJsonName;
+    }
+
+    let name = uniqueNamesGenerator({
+        dictionaries: [ungColors, adjectives, animals],
+        separator: "-",
+        style: "lowerCase",
+        length: 3,
+    });
+    if (process.env["CI"] !== "true") {
```

Comment:
use `isCI()` instead
https://github.com/Genez-io/genezio/blob/dev/src/utils/process.ts#L47

```suggestion
    if (!isCI()) {
```